certified-connectors/Verified/apiDefinition.swagger.json

```diff
@@ -0,0 +1,2681 @@
+          }
+        ],
+        "responses": {
+          "201": {
+            "description": "OK",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "uid": {
+                  "type": "string"
+                }
+              }
+            }
+          }
+        },
+        "x-ms-visibility": "important"
+      }
+    },
+    "/envelope-descriptors/default": {
+      "get": {
+        "operationId": "GetEnvelopeDescriptorsDefault",
+        "summary": "Get default descriptor",
+        "description": "Gets the default descriptor for envelopes",
+        "parameters": [
+          {
+            "$ref": "#/parameters/Token"
+          },
+          {
+            "$ref": "#/parameters/ContentType"
+          },
+          {
+            "$ref": "#/parameters/XNamespace"
+          },
+          {
+            "$ref": "#/parameters/Filters"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/Descriptor"
+            }
+          }
+        },
+        "x-ms-visibility": "advanced"
+      }
+    },
+    "/envelopes/{envelope-id}/jobs/send.notification": {
+      "post": {
+        "operationId": "PostEnvelopesEnvelopIdJobsSendNotification",
+        "summary": "Send notification",
+        "description": "Sends a notification to recepients of a given envelope,",
+        "parameters": [
+          {
+            "$ref": "#/parameters/Token"
+          },
+          {
+            "$ref": "#/parameters/ContentType"
+          },
+          {
+            "$ref": "#/parameters/XNamespace"
+          },
+          {
+            "$ref": "#/parameters/EnvelopeId"
+          },
+          {
+            "name": "body",
+            "in": "body",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "envelope": {
+                  "type": "object",
+                  "title": "Envelope",
+                  "properties": {
+                    "greeting": {
+                      "type": "string",
+                      "title": "Greeting"
+                    }
+                  }
+                },
+                "recipient": {
+                  "type": "object",
+                  "title": "Recipient",
+                  "properties": {
+                    "id": {
+                      "type": "string",
+                      "description": "The ID of the recipient the notification should be send to.",
+                      "title": "Recipient Id",
+                      "x-ms-visibility": "important"
+                    }
+                  },
+                  "required": [
+                    "id"
+                  ]
+                }
+              },
+              "required": [
+                "recipient"
+              ]
+            }
+          }
+        ],
+        "responses": {
+          "202": {
```

Comment:
As stated in the API documentation this endpoint does not return any information in the body either
https://verified-dev.api-docs.io/v2.1/envelopes/post-envelopes-param1-jobs-send-notification